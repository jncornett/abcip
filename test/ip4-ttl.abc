d ( stack="eth:ip4:udp" )

c ( ip4:a="76.0.0.10", b="4.4.4.10" )

a( ip4:tos=0x10, ttl=63; pay=
   "1234567890123456789012345678901234567890123456789012345678901234")
