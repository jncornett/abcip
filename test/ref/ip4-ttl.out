IP (tos 0x10, ttl 63, id 1, offset 0, flags [none], proto UDP (17), length 92)
    76.0.0.10.48620 > 4.4.4.10.8: UDP, length 64
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4510  ..............E.
	0x0010:  005c 0001 0000 3f11 2769 4c00 000a 0404  .\....?.'iL.....
	0x0020:  040a bdec 0008 0048 4ccd 3132 3334 3536  .......HL.123456
	0x0030:  3738 3930 3132 3334 3536 3738 3930 3132  7890123456789012
	0x0040:  3334 3536 3738 3930 3132 3334 3536 3738  3456789012345678
	0x0050:  3930 3132 3334 3536 3738 3930 3132 3334  9012345678901234
	0x0060:  3536 3738 3930 3132 3334                 5678901234
